### PR TITLE
Issue #3123861 by Kingdutch: Image label is visible on basic page tea…

### PR DESCRIPTION
**Pull request by Kingdutch**
…sers
<h2>Problem</h2>
The view configuration shows the label for the image of basic pages and book pages in teasers and heros.

<h2>Solution</h2>
Update the view configuration

## Issue tracker
https://www.drupal.org/project/social/issues/3123861

## How to test
- [ ] View a basic page in the search results
- [ ] Create (with hero image) and view the detail page of a book page.

## Screenshots
If this Pull Request makes visual changes then please include some screenshots that show what has changed here. 

## Release notes
The label for the image field is no longer visible in teasers for basic pages.

### CODE REVIEW

**Comment by robertragas** [comment]
I have updated this PR with additional setting changes for the social_book which had the same problem.

**Comment by ronaldtebrake** [comment]
Ensure update_helper module is enabled by adding dependencies to social_core 8805 (for those sites that come from a very old version and might skip numbers it's very important :) )

**Comment by Kingdutch** [comment]
Declared the dependency for `social_book_8802`. `social_page_8801` already had a dependency on `social_core_8805` so this isn't needed for `social_page_8802`

File: config/optional/core.entity_view_display.node.book.teaser.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ content:
   field_book_image:
     type: image
     weight: 0
-    label: above
+    label: hide
```
Comment (ronaldtebrake) [inline config/optional/core.entity_view_display.node.book.teaser.yml:26]: shouldn't this be hidden? 🤔

Comment (Kingdutch) [reply config/optional/core.entity_view_display.node.book.teaser.yml]: @ronaldtebrake Not sure what happened there, fixed it :)

**Comment by ronaldtebrake** [comment]
Travis was failing due to still testing old core version. Will update this in a separate branch. Manual tests and green checks before made me decide to accept it.

**Comment by ronaldtebrake** [comment]
Also cherry-picked to 8.x

**Pull request merged**